mautic/mautic
# Reports not equal expression include nullable values

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for 4.0
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
Reports not return nullable values for not equal.



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contact with email address
3. Create reports with contacts and with filter address1 not equal xxx
4. You should see created contact from step 2, but it's not
5. After fix you see created contact in result




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10043"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
